README.md

```diff
@@ -2,7 +2,211 @@
 
 # Dockerhub
 
-Images can be found here: https://hub.docker.com/_/microsoft-azure-functions-base
+## V3 Images
+
+
+#### Dotnet
+
+`mcr.microsoft.com/azure-functions/dotnet`
+
+Linux amd64 Tags
+
+| Tags     | Dockerfile     | OS Version    |
+| -------- | -------------- | ------------- |
+| `3.0`      | [Dockerfile]() |   Debian 10   |
+| `3.0-slim` | [Dockerfile]() |   Debian 10   |
+| `3.0-appservice`, `3.0-dotnet3-appservice`, `3.0-appservice-quickstart` | [Dockerfile]() | Debian 10 |
+
+Linux arm32v7 Tags
+
+| Tags               | Dockerfile     | OS Version    |
+| ------------------ | -------------- | ------------- |
+| `3.0-arm32v7`        | [Dockerfile]() | Debian 10     |
+| `3.0-bionic-arm32v7` | [Dockerfile]() | Ubuntu 18.04  |
+
+Windows Server
+
+| Tags                | Dockerfile     | OS Version          |
+| ------------------- | -------------- | ------------------- |
+| `3.0-nanoserver-1809` | [Dockerfile]() | Windows Server 1809 |
+
+#### Node
+
+`mcr.microsoft.com/azure-functions/node`
+
+Linux amd64 Tags
+
+| Tags                      | Dockerfile      | OS Version   |
+| ------------------------- | --------------- | ------------ |
+| `3.0`, `3.0-node10`           | [Dockerfile]()  | Debian 10    |
+| `3.0-slim`, `3.0-node10-slim` | [Dockerfile]()  | Debian 10    |
+| `3.0-appservice`, `3.0-appservice-quickstart`, `3.0-node10-appservice`, `3.0-node10-appservice-quickstart` | [Dockerfile]() | Debian 10 |
+| `3.0-node12`                | [Dockerfile]()  | Debian 10    |
+| `3.0-node12-slim`           | [Dockerfile]()  | Debian 10    |
+| `3.0-node12-appservice`, `3.0-node12-appservice-quickstart` | [Dockerfile]() | Debian 10 |
+
+#### Powershell
+
+`mcr.microsoft.com/azure-functions/powershell`
+
+Linux amd64 Tags
+
+| Tags                           | Dockerfile      | OS Version   |
+| ------------------------------ | --------------- | ------------ |
+| `3.0`, `3.0-powershell6`           | [Dockerfile]()  | Debian 10    |
+| `3.0-slim`, `3.0-powershell6-slim` | [Dockerfile]()  | Debian 10    |
+| `3.0-appservice`, `3.0-appservice-quickstart`, `3.0-powershell6-appservice`, `3.0-powershell6-appservice-quickstart` | [Dockerfile]() | Debian 10 |
+
+#### Java
+
+`mcr.microsoft.com/azure-functions/java`
+
+Linux amd64 Tags
+
+| Tags                     | Dockerfile      | OS Version   |
+| ------------------------ | --------------- | ------------ |
+| `3.0`, `3.0-java8`           | [Dockerfile]()  | Debian 10    |
```

Comment:
Really small thing there is some of them has `` and other are not.
Please can you guide how is the [Dockerfile]() works? As the link did not work for me when I tried.

Reply:
Will add `` to the rest of the image names. 

And yes the link doesn't work now, I still need to put the full path from github there for it to work